[2022-06-08 06:23:53,206] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.drop_order_table manual__2022-06-08T06:18:40.397380+00:00 [queued]>
[2022-06-08 06:23:53,221] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.drop_order_table manual__2022-06-08T06:18:40.397380+00:00 [queued]>
[2022-06-08 06:23:53,222] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-08 06:23:53,223] {taskinstance.py:1357} INFO - Starting attempt 5 of 6
[2022-06-08 06:23:53,224] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-08 06:23:53,244] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): drop_order_table> on 2022-06-08 06:18:40.397380+00:00
[2022-06-08 06:23:53,251] {standard_task_runner.py:52} INFO - Started process 15274 to run task
[2022-06-08 06:23:53,255] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'hands_on_test', 'drop_order_table', 'manual__2022-06-08T06:18:40.397380+00:00', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp7mky6qt8', '--error-file', '/tmp/tmpyamlpu9h']
[2022-06-08 06:23:53,257] {standard_task_runner.py:80} INFO - Job 182: Subtask drop_order_table
[2022-06-08 06:23:53,336] {task_command.py:369} INFO - Running <TaskInstance: hands_on_test.drop_order_table manual__2022-06-08T06:18:40.397380+00:00 [running]> on host b79bd65fcc57
[2022-06-08 06:23:53,444] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hands_on_test
AIRFLOW_CTX_TASK_ID=drop_order_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-08T06:18:40.397380+00:00
AIRFLOW_CTX_TRY_NUMBER=5
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-08T06:18:40.397380+00:00
[2022-06-08 06:23:53,457] {base.py:68} INFO - Using connection ID 'postgres_db' for task execution.
[2022-06-08 06:23:53,472] {dbapi.py:208} INFO - Running statement: 
DROP TABLE IF EXISTS orders;, parameters: None
[2022-06-08 06:23:53,477] {postgres.py:94} INFO - NOTICE:  table "orders" does not exist, skipping

[2022-06-08 06:23:53,496] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hands_on_test, task_id=drop_order_table, execution_date=20220608T061840, start_date=20220608T062353, end_date=20220608T062353
[2022-06-08 06:23:53,549] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-08 06:23:53,648] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
